2019/swift-cheatsheet/readme.md

```diff
@@ -405,6 +405,18 @@ var namesArray = [String]()
 
 Remember, Swift is a strongly-typed language so you need to specify the data type being stored in the array at the time the array is created.
 
+The above method uses Swift's type-inference engine to determine that the value is an array of strings, as we haven't specified it with the type declaration. You can do this in two ways - longhand:
```

Comment:
Thanks Jack. Could you reword just to simply "There are also two other ways you can declare an empty array of a particular type in Swift"